test/unit/models/tasks.test.js

```diff
@@ -303,12 +303,12 @@ describe("tasks", function () {
       overdueTask.endsOn = Date.now() / 1000 + 24 * 60 * 60 * 7;
       await tasks.updateTask(overdueTask);
       const usersWithOverdueTasks = await tasks.getOverdueTasks(days);
-      expect(usersWithOverdueTasks.length).to.be.equal(5);
+      expect(usersWithOverdueTasks.length).to.be.equal(8);
     });
 
     it("should return all users which have overdue tasks if days is not passed", async function () {
       const usersWithOverdueTasks = await tasks.getOverdueTasks();
-      expect(usersWithOverdueTasks.length).to.be.equal(4);
+      expect(usersWithOverdueTasks.length).to.be.equal(7);
```

Comment:
why this change?

Reply:
I increased the users in the mock data to include the required data for the new API. This addition increased the user count by 3. While verifying the test cases, I noticed some were failing due to the changes in mock data, so I updated and fixed them accordingly.

Reply:
Why are we modifying the tests which are already working? Can't we create another variable for users mock data?

Reply:
I can create a separate mock data. My initial approach of modifying the existing fixtures was based on the pattern I observed in our codebase, where previous additions were made to the same fixtures file to keep mock data centralised and easier to maintain.

Reply:
Have added the changes @pankajjs